packages/hl7v2-server/tsconfig.json

```diff
@@ -0,0 +1,17 @@
+{
+  "compilerOptions": {
+    "target": "es2020",
+    "module": "commonjs",
+    "outDir": "./dist",
+    "rootDir": "./src",
+    "strict": true,
+    "esModuleInterop": true,
+    "skipLibCheck": true,
+      "resolveJsonModule": true,
+      "sourceMap": true,
+      "inlineSources": true,
+    "forceConsistentCasingInFileNames": true
+  },
+  "include": ["src/**/*"],
+  "exclude": ["node_modules", "dist"]
```

Comment:
something I learned after we built ours, is that `exclude` is based on the things we included, so no need to exclude `node_modules` and `dist`, since they're not inside `src` - [reference](https://www.typescriptlang.org/tsconfig/#exclude).

Reply:
addressed